### PR TITLE
selectlist property type

### PR DESCRIPTION
**Pull request by jberends**
* options getter and setter
* assertions
* tests (WIP)

### CODE REVIEW

**Comment by jberends** [comment]
* refactored naming of the files for attachment property and selectlist property
* added representation to the partset
* updated all tests for the select listproperty

**Comment by jberends** [comment]
@wilcoschoneveld

**Comment by jberends** [comment]
@wilcoschoneveld

File: pykechain/client.py [hunk 1]
```diff
@@ -62,10 +60,40 @@ def __repr__(self):
...
+            KECHAIN_PASSWORD=...
+
+
+        >>> client = Client().from_env()
+
+        """
+        env.read_envfile(env_filename)
         client = cls(url=env('KECHAIN_URL'))
-        client.login(token=env('KECHAIN_TOKEN'))
+        if env('KECHAIN_TOKEN'):
```
Comment (wilcoschoneveld) [inline pykechain/client.py]: you can login with token, username AND password; `client.login` already puts priority on token.

Comment (jberends) [inline pykechain/client.py]: That is indeed the case and known.

**Pull request merged**